### ISSUE TITLE
Script unblocks ads

### ISSUE THREAD

**Issue by YoueG** [zerodytrash/Simple-YouTube-Age-Restriction-Bypass#11]
With the script enabled, ads usually blocked are unblocked

Configuration:
- Windows 10
- Firefox 89
- uBlock origin

**Comment by YoueG** [comment]
Closing issue as it seems to be a duplicate of : https://github.com/zerodytrash/Simple-YouTube-Age-Restriction-Bypass/issues/6
